thirdparty/patchmatchnet/datasets/data_io.py

```diff
@@ -0,0 +1,93 @@
+"""MVSDataset io utils
+    reference: https://github.com/FangjinhuaWang/PatchmatchNet
+
+"""
+import sys
+from typing import Tuple
+
+import re
+import numpy as np
+
+
+def read_pfm(filename: str) -> Tuple[np.ndarray, float]:
+    """Read a depth map from a .pfm file
+
+    Args:
+        filename: .pfm file path string
+
+    Returns:
+        data: array of shape (H, w, C) representing loaded heat map
+        scale: float to recover heat map pixel values to actual probabilities
```

Comment:
why does the depth map contain probabilities? is it not save as "winner-take-all"?